packages/rollup-full-node/test/integration/transpiler-integration.spec.ts

```diff
@@ -0,0 +1,95 @@
+import '../../../rollup-dev-tools/test/setup'
+/* External Imports */
+import {
+  getLogger,
+  remove0x,
+  bufToHexString,
+  hexStrToBuf,
+} from '@eth-optimism/core-utils'
+import {
+  Address,
+  formatBytecode,
+  bufferToBytecode,
+} from '@eth-optimism/rollup-core'
+
+/* Internal Imports */
+
+import * as SimpleStorage from '../contracts/build/transpiled/SimpleStorage.json'
+import * as SimpleCaller from '../contracts/build/transpiled/SimpleCaller.json'
+import * as SelfAware from '../contracts/build/transpiled/SelfAware.json'
+import * as CallerGetter from '../contracts/build/transpiled/CallerGetter.json'
+import * as OriginGetter from '../contracts/build/transpiled/OriginGetter.json'
+import * as CallerReturner from '../contracts/build/transpiled/CallerReturner.json'
+import * as TimeGetter from '../contracts/build/transpiled/TimeGetter.json'
```

Comment:
Btw one thing I forgot to mention is that technically if we were keeping to convention there'd be a comment for `Contract Imports` which has all of these as opposed to internal imports, even though of course contract imports are internal. But just a note that's the convention